platforms/nuttx/src/bootloader/microchip/mpfs/main.c

```diff
@@ -719,7 +727,9 @@ arch_do_jump(const uint32_t *app_base)
 
 	/* PX4 on hart 2 */
 #if CONFIG_MPFS_HART2_ENTRYPOINT != 0xFFFFFFFFFFFFFFFF
-	_alert("Jump to PX4 0x%lx\n", (uint64_t)app_base);
+	bool use_sbi = info_bits & INFO_BIT_USE_SBI ? true : false;
+	_alert("Jump to PX4 0x%lx %s\n", (uint64_t)app_base, use_sbi ? "via SBI" : "");
+	mpfs_set_use_sbi(2, use_sbi);
 	mpfs_set_entrypt(2, (uintptr_t)app_base);
```

Comment:
There is an extern mpfs_set_entrypt somewhere in the file, can you please remove that ?

Reply:
yes, done!